physics/cu_gf_driver.F90

```diff
@@ -258,10 +331,10 @@ subroutine cu_gf_driver_run(ntracer,garea,im,km,dt,flag_init,flag_restart,&
 ! these should be coming in from outside
 !
 !    cactiv(:)      = 0
-!    cactiv_m(:)    = 0
```

Comment:
This line should not be removed

Reply:
That line is commented out. It could be replaced by ASCII art of an elephant and the results would not change.

Reply:
Yes, but if we turn off aerosol-awareness again this needs to be turned back on. I guess I could manually add it back in if that happens

Reply:
I put the comment back in.